### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by stord-engineering-account**
:robot: I have created a release *beep* *boop*
---

<details><summary>1.2.1</summary>

### Features

* [SRE-830] Add trivy docker scan configuration and matrix outputs ([#7](https://github.com/stordco/actions-trivy/issues/7)) ([98c14a2](https://github.com/stordco/actions-trivy/commit/98c14a203eff50931a79d42b2abe0d7ab0990ec3))
* SRE-955 support skipping trivy db update ([#13](https://github.com/stordco/actions-trivy/issues/13)) ([486ae16](https://github.com/stordco/actions-trivy/commit/486ae16ff6ec2ade80cc221cf6119b92f09f3e48))

### Miscellaneous Chores

* release 1.2.1 ([#12](https://github.com/stordco/actions-trivy/issues/12)) ([d39695a](https://github.com/stordco/actions-trivy/commit/d39695a40073ca5e5db1e3eb7f6a216b8be7a63a))
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

[SRE-830]: https://stord.atlassian.net/browse/SRE-830?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by msutkowski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ For default usage:
     uses: stordco/actions-trivy@v1
     with:
         scan-type: image
-        image-ref: gcr.io/stord-ci/app-base:2024.06.25_d5cd08e
+        image-ref: gcr.io/stord-ci/app-base:1.2.1_d5cd08e
```
Comment (msutkowski) [inline README.md:60]: 🤔 🤔 🤔 🤔  weird.

Comment (btkostner) [reply README.md:60]: Bad change. It's updating a reference the stord-ci app base, and not to the action-trivy version.

Comment (msutkowski) [inline README.md:60]:
```suggestion
        image-ref: gcr.io/stord-ci/app-base:2024.06.25_d5cd08e
```

File: README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ For default usage:
...
```
Comment (msutkowski) [inline README.md:73]:
```suggestion
        image-ref: gcr.io/stord-ci/app-base:2024.06.25_d5cd08e
```

**Review by msutkowski — APPROVED** [review]

**Pull request merged**